nixos/modules/services/networking/openvpn3.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.openvpn3;
+in
+{
+  ###### interface
+
+  options.services.openvpn3.client = {
+    enable = mkEnableOption "openvpn3 client. It might override your DNS servers; if that causes you problems, look into \"--dns-setup-disabled true\" or \"--dns-fallback-google true\" options in \"openvpn3 config-manage\" command";
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.client.enable {
+    services.dbus.packages = [
+      pkgs.openvpn3
+    ];
+
+    users.users.openvpn = {
+      isSystemUser = true;
+      group = "openvpn";
```

Comment:
I don't understand this suggestion. Shouldn't `openvpn` user just be a homeless system user?

Reply:
I think you're right, the default is `/var/empty`. It's more like, I don't understand the benefit of assigning a home to the system user. Would that solve the issue of configs currently not being saved?

Reply:
depends where it is writing that. It could be.

Reply:
It appears that the resulting derivation has `var` directory, which it uses instead of the system's `var` directory. That's probably why configurations don't get saved: you can't really write to a Nix store. How would I go about solving that issue?